sources/packages/web/src/views/Home.vue

```diff
@@ -1,13 +1,31 @@
 <template>
-  <div>Home - Student Dashboard</div>
-  <div>
-    You are here because Account information for this user exists within SABC
+  <div style="padding-left: 10px">
```

Comment:
The current framework has a set of classes to be used for spacing. The same as line 5.
This is a real minor comment because we are gonna change the framework, but it could be handy to use the already existing classes.
https://primefaces.org/primevue/showcase/#/spacing

Reply:
FIXED